build.gradle

```diff
@@ -67,20 +55,6 @@ dependencies {
     api "com.exactpro.th2:grpc-service-generator:${serviceGeneratorVersion}"
 }
 
-sourceSets {
-    main {
-        proto {
-            srcDirs 'src/main/proto'
-        }
-        java {
-            srcDirs 'src/gen/main/java', 'src/gen/main/grpc', 'src/gen/main/services/java/src'
-        }
-        resources {
-            srcDirs 'src/gen/main/services/java/resources'
-        }
```

Comment:
This must be left and destination corrected to the place where the updated protobuf generator places the generated source